plugins/services/src/js/containers/services/ServicesContainer.js

```diff
@@ -151,6 +153,9 @@ class ServicesContainer extends React.Component {
   componentDidMount() {
     DCOSStore.addChangeListener(DCOS_CHANGE, this.onStoreChange);
 
+    // Don't block the whole screen if Mesos state is not there
+    MesosStateStore.addChangeListener(MESOS_STATE_CHANGE, function () {});
```

Comment:
We need to subscribe to the store so that we get tasks for frameworks.
Notice that we're using `MesosStateStore` in Framework.js